src/wallet/wallet.cpp

```diff
@@ -2377,11 +2379,12 @@ void CWallet::ResendWalletTransactions()
         // Relay transactions
         for (std::pair<const uint256, CWalletTx>& item : mapWallet) {
             CWalletTx& wtx = item.second;
-            // only rebroadcast unconfirmed txes older than 5 minutes before the
-            // last block was found
+            // Attempt to rebroadcast all txes more than 5 minutes older than
+            // the last block. SubmitMemoryPoolAndRelay() will not rebroadcast
+            // any confirmed or conflicting txs.
```

Comment:
Why not move the coinbase check here then?

Reply:
I agree that would make sense - the other callers of `SubmitMemoryPoolAndRelay()` do checks before calling into `SubmitMemoryPoolAndRelay()`. However, this PR is just to restore previous behaviour. There are a few other PRs in progress on the node/wallet interface, so let's keep the existing behaviour and look for ways to tidy it up later.